app/views/your_contact_details/your_contact_details.scala.html

```diff
@@ -85,6 +85,9 @@
     ))
 
     <p class="govuk-body govuk-!-padding-bottom-9" id="contact-report-change-link">
+
+        @messages("cf.contact-details.report-change-pre-link-text")
```

Comment:
Could you please make use of preLinkMessage of newTabLink component and add an optional postLinkMessage and make use of that as well? Would be great if you could pass the style classes as arguement

You could see the example in customs-cash-account-frontend

Also add the tests as well.